CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+
+## [1.2.12](https://github.com/ably/ably-flutter/tree/v1.2.12)
+
+[Full Changelog](https://github.com/ably/ably-flutter/compare/v1.2.11...v1.2.12)
+
+**Fixed bugs:**
+
+- Android: `IllegalStateException` in `Crypto` `CBCCipher`'s `decrypt` method [\#314](https://github.com/ably/ably-flutter/issues/314), fixed by updating to `ably-android` version 1.2.11 [\#319](https://github.com/ably/ably-flutter/pull/319) ([QuintinWillison](https://github.com/QuintinWillison))
```

Comment:
`updating to `ably-android`
Is it suppose to be ably-flutter?

Reply:
No. This point refers to the `ably-android` dependency.

Reply:
I see @AndyNicks has now approved this PR, so resolving this conversation.